### ISSUE TITLE
Rule addition: "dollar-variable-empty-line-before" ignore: "after-dollar-variable"

### ISSUE THREAD

**Issue by Traverse** [kristerkari/stylelint-scss#105]
Would like to see this addition because sometimes I'd like to group variables which currently isn't possible with the available options.

**Comment by dryoma** [comment]
Hi @Traverse 

A legit suggestion. Just to clarify: do you think the #94 's  `between-groups`, if/when implemented, work for your case?

**Comment by Traverse** [comment]
I think it should work for my case, it mostly depends on how it checks if something is a group or not because sometimes you might want to group variables even if they don't have a prefix that is similar etc.

Might be that my option is simpler / enough in most cases.

**Comment by jeddy3** [comment]
In `stylelint@7.8.0` [we standardised](https://github.com/stylelint/stylelint/issues/1663#issue-165965730) the options of the `*-empty-line-before` rules to use `after-*`. As such, I think adding an `"ignore": ["after-dollar-variable"]` option would be the idiomatic solution to this issue.

**Comment by ArmorDarks** [comment]
Would be great to see this feature, since as for now `scss/dollar-variable-empty-line-before` indeed doesn't allow grouping in such style:

```scss
code {
  $test: 'value';
  $test2: 'value';

  $otherVar: 'value';
  $otherVar2: 'value';

  padding: 'value';
}
```
